src/richie/apps/courses/templates/courses/cms/fragment_organization_main_logo.html

```diff
@@ -0,0 +1,17 @@
+{% load cms_tags static %}{% spaceless %}
+{% if current_page.publisher_is_draft %}
+    {% if organization.check_publication is True %}
+        {% show_placeholder "logo" organization.public_extension.extended_object as logo %}
+    {% else %}
+        {% show_placeholder "logo" organization.extended_object as logo %}
+    {% endif %}
+{# If the current page is the published version, show only the organizations that are published #}
+{% elif organization.check_publication is True %}
+    {% show_placeholder "logo" organization.public_extension.extended_object as logo %}
+{% endif %}
+{% if logo %}
+    {{ logo }}
+{% else %}
+    <img src="{% static "images/samples/main-organization.png" %}" alt="">
```

Comment:
should still have a alt IMO. 
Proposal:
alt="{% trans "organization logo placeholder" %}"